File: jApp/src/main/java/j/s/yarlykov/ui/fragmentbased/InfoActivityFr.java
package j.s.yarlykov.ui.fragmentbased;

import android.content.Context;
import android.content.Intent;
import android.os.Bundle;
import android.support.annotation.Nullable;
import android.support.v7.app.ActionBar;
import android.support.v7.app.AppCompatActivity;

import j.s.yarlykov.R;

public class InfoActivityFr extends AppCompatActivity {

    public static void start(Context context) {
        context.startActivity(new Intent(context, InfoActivityFr.class));
    }

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        ActionBar ab = getSupportActionBar();
        if(ab != null) {
            ab.hide();
        }

        setContentView(R.layout.activity_info_fr);
    }
}

File: kApp/src/main/java/k/s/yarlykov/ui/fragmentbased/InfoActivityFr.kt
package k.s.yarlykov.ui.fragmentbased

import android.content.Context
import android.content.Intent
import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import android.view.Window
import k.s.yarlykov.R

class InfoActivityFr: AppCompatActivity() {

    companion object {
        fun start(context: Context) {
            val intent = Intent(context, InfoActivityFr::class.java)
            context.startActivity(intent)
        }
    }

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        supportActionBar?.hide()
        setContentView(R.layout.activity_info_fr)
    }
}